apps/content_types/content_types.py

```diff
@@ -562,7 +562,7 @@ def apply_schema(item):
     :param item: item to apply schema to
     """
     # fields that can be added to article without being added to CP eg: using widgets
-    allowed_keys = ["attachments", "refs"]
+    allowed_keys = ["attachments", "refs", "place"]
```

Comment:
what about `organisation` ?

Reply:
we don't need the organisation or person because if it's not in the content profile then we do not need them anywhere, However that's not the case for place, if place is not present in the content profile then also we need it.

Reply:
imatrics return organisation don't they? and we need those in the output

Reply:
I would rather add all those imatrics fields now so we don't have to create core release later for that